packages/docs/examples/va-checkbox/Error.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <va-checkbox
+      v-model="value"
+      error
+      :error-messages="['Error message', 'Another error message', 'One more error message']"
```

Comment:
No need to showcase multiple error messages, one is good. We'll likely cover these on validation page.